arcgis-ios-sdk-samples/Layers/Change sublayer renderer/README.md

```diff
@@ -1,16 +1,40 @@
 # Change sublayer renderer
 
-This sample demonstrates how you to change renderer of a sublayer in map image layer
+Apply a renderer to a sublayer.
 
-## How the app works
+![Change sublayer renderer sample](change-sublayer-renderer.png)
 
-Tap on `Apply renderer` button to apply a class breaks renderer based on population for each county. Tap on the `Reset` button to reset to original renderer.
+## Use case
 
-![](image1.png)
+A layer showing animal populations contains sublayers for different species. A renderer could be applied which gives each sublayer a different color, so that populations of each species can be compared visually.
+
+## How to use the sample
+
+Tap on "Apply renderer" button to apply a class breaks renderer based on population for each county. Tap on the "Reset" button to reset to original renderer.
```

Comment:
```suggestion
Tap on "Apply renderer" button to apply a class breaks renderer based on population for each county. Tap on "Reset" button to reset to original renderer.
```
I'm always confused by the usage of articles in English... 😓  Correct me if I made it wrong.